### PR TITLE
feat: add ubi mapping for base image recommendation

### PR DESCRIPTION
**Pull request by xieshenzh**
Create image mapping for recommending UBI and update the html report template.

Jira: https://issues.redhat.com/browse/APPENG-2274

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -35,4 +35,13 @@ quarkus.rest-client.segment-api.url=https://api.segment.io/
 quarkus.management.enabled=true
 quarkus.http.limits.max-body-size=4G
 
-trustedcontent.recommended.ubi=pkg:oci/registry.access.redhat.com/ubi9/ubi@latest
+trustedcontent.recommendation.ubi.purl.ubi9=pkg:oci/ubi@sha256:f5983f7c7878cc9b26a3962be7756e3c810e9831b0b9f9613e6f6b445f884e74?repository_url=registry.access.redhat.com/ubi9/ubi&tag=9.3-1552&arch=amd64
+trustedcontent.recommendation.ubi.catalogurl.ubi9=https://catalog.redhat.com/software/containers/ubi9/ubi/615bcf606feffc5384e8452e?architecture=amd64&image=65a82982a10f3e68777870b5f
```
Comment (xieshenzh) [inline src/main/resources/application.properties:39]: @olavtar I moved the configurations for the links to application.properties

**Comment by xieshenzh** [comment]
@ruromero @olavtar I don't have access to the Konflux pipeline.
Could you take a look and let me know why it failed?

**Review by ruromero — APPROVED** [review]
LGTM

**Pull request merged**